Editor/EditorCore/StripProBuilderScripts.cs

```diff
@@ -125,5 +118,16 @@ public static void DoStrip(ProBuilderMesh pb)
             }
             catch {}
         }
+
+        static void DestroyProBuilderMeshAndDependencies(GameObject go, ProBuilderMesh pb)
+        {
+            DestroyImmediate(pb);
+
+            if (go.TryGetComponent(out Entity entity))
+                DestroyImmediate(entity);
+
+            if (go.TryGetComponent(out ShapeComponent shapeComponent))
+                DestroyImmediate(shapeComponent);
```

Comment:
is this missing spline and polyshape components?

Reply:
also, can this use the same function as what we have in `UnityScenePostProcessor`?

Reply:
I tried to put the scripts cleaning in a single method. 
However, each place using script stripping is doing this in a different way, which complicates this...

I also updated script stripping when exporting prefab and meshes.
They all call now the same method to ease future stripping extensions is needed.